extensions/amp-story/0.1/amp-story-store-service.js

```diff
@@ -47,12 +48,14 @@ export const StateProperty = {
 
   // App States.
   BOOKEND_STATE: 'bookendstate',
+  CURRENT_PAGE: 'currentpage',
 };
 
 
 /** @private @const @enum {string} */
 export const Action = {
   TOGGLE_BOOKEND: 'togglebookend',
+  PAGE_CHANGE: 'pagechange',
```

Comment:
``PAGE_CHANGE`` makes it sound like an event, when it really is an action. What do you think of a more direct ``CHANGE_PAGE``?